rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -20,23 +18,14 @@ import monix.eval.Task
 
 import scala.collection.immutable
 
-class Runtime private (
-    val reducer: Reduce[Task],
-    val replayReducer: Reduce[Task],
-    val space: ISpace[Channel,
-                      BindPattern,
-                      ListChannelWithRandom,
-                      ListChannelWithRandom,
-                      TaggedContinuation],
-    val replaySpace: ReplayRSpace[Channel,
-                                  BindPattern,
-                                  ListChannelWithRandom,
-                                  ListChannelWithRandom,
-                                  TaggedContinuation],
-    val costAccountingPure: CostAccountingAlg[Task],
-    val costAccountingReplay: CostAccountingAlg[Task],
-    var errorLog: ErrorLog,
-    val context: Context[Channel, BindPattern, ListChannelWithRandom, TaggedContinuation]) {
+class Runtime private (val reducer: Reduce[Task],
+                       val replayReducer: Reduce[Task],
+                       val space: Space[ISpace],
+                       val replaySpace: Space[ReplayRSpace],
+                       val costAccountingPure: CostAccountingAlg[Task],
+                       val costAccountingReplay: CostAccountingAlg[Task],
+                       var errorLog: ErrorLog,
+                       val context: Store[Context]) {
```

Comment:
IDK if renaming `Context` to `Store` is a good change.

Reply:
We can change the name. The current names were supposed to convey the meaning as follows:

`Repository.Space` - the set of parameters that Runtime uses for ISpace, RSpace, ReplayRSpace, and anything else that needs that exact quintuple of type parameters

`Repository.Store` - the set of parameters that Runtime uses for ISpace, and anything else that needs that exact quadruple of type parameters. E.g. Context.

I was thinking of using `CPARK` and `CPAK`, but dismissed those as cryptic.

Maybe there's a naming convention for type aliases like these?
`type FixParams[A[_]] = A[Int]` 
`type FixParams2[A[_, _]] = A[Int, String]`

Reply:
Or, we could rename them to MyFoo, which would then excuse writing `MySpace[ISpace]` :stuck_out_tongue: (remember MySpace?)

Reply:
One more variant:
`StoreParams[IStore]`
`StoreParams[Context]`
`SpaceParams[Rspace]`